File: ruby/delete_attachment.rb
require 'uri'
require 'net/http'

url = URI("https://graph.microsoft.com/v1.0/me/events/%7Bid%7D/attachments/%7Bid%7D")

http = Net::HTTP.new(url.host, url.port)
http.use_ssl = true
http.verify_mode = OpenSSL::SSL::VERIFY_NONE

request = Net::HTTP::Delete.new(url)
request["content-type"] = 'application/json'
request.body = "{ }"

response = http.request(request)
puts response.read_body

File: csharp/delete_attachment.cs
var client = new RestClient("https://graph.microsoft.com/v1.0/me/events/%7Bid%7D/attachments/%7Bid%7D");
var request = new RestRequest(Method.DELETE);
request.AddHeader("content-type", "application/json");
request.AddParameter("application/json", "{ }", ParameterType.RequestBody);
IRestResponse response = client.Execute(request);
